### DISCUSSION TITLE
Hi, Have you guys implement the BSSRDF

### DISCUSSION THREAD

**Discussion by dandingol03** [google/filament#4742] [Q&A]
Hi
   i see the `BSSRDF` is marked as todo, so when does filament support it?

**Comment by romainguy** [comment]
Which BSSRDF are you talking about? Filament already supports an approximation of subsurface scattering, but we want to add proper support for skin, etc. The existing todos are just for the documentation.

**Reply by dandingol03** [reply]
Hi, thanks for your reply. I mean the BSSRDF which the paper `A Practical Model for Subsurface Light Transport` had mentioned. Please show me the code about the subsurface scattering mentioned above if it is convenient to you~
